tokio/src/util/usdt.rs

```diff
@@ -0,0 +1,142 @@
+cfg_rt! {
```

Comment:
I think we probably want to rename this to `usdt/mod.rs` to keep in the style of the rest of the project.